lib/versioning/utils.js

```diff
@@ -0,0 +1,36 @@
+const VSConst = require('./constants').VersioningConstants;
+
+const MSTLEN = VSConst.VERSIONING_AFFIXES.MASTER_VERSION_PREFIX.length;
+const VERLEN = VSConst.VERSIONING_AFFIXES.OTHER_VERSION_PREFIX.length;
+const SEP = VSConst.VERSIONING_AFFIXES.SEPARATOR;
+
+module.exports.VersioningUtils = {
+    // using explicit encode and decode functions for version object to enable
+    // future extensions, such as attatching external information to the string
+    // value from request instead of having to parse and stringify the value
+    // Example: "prefixlen_{prefix}{value_from_request}"
+    encodeVersion: version => JSON.stringify(version),
+    decodeVersion: version => JSON.parse(version),
```

Comment:
Will update. But personally I think crashing should be the intended result, rather than covering it and messing with the errors later. After all, it should be an internal error if there is anything wrong with the format here, in such case we should restart rather than trying to make more errors.